### PR TITLE
create maintenance glossary and generation script

### PR DESCRIPTION
**Pull request by Caitlin2694**

### CODE REVIEW

**Review by mhodki — APPROVED** [review]

File: maintenance/scripts/glossary_template.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+  <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      <meta name="generator" content="PhpSpreadsheet, https://github.com/PHPOffice/PhpSpreadsheet">
+      <title>Untitled Spreadsheet</title>
```
Comment (mattys101) [inline maintenance/scripts/glossary_template.html:6]:
```suggestion
      <title>IOF MRO Glossary</title>
```
Or similar.

File: maintenance/scripts/glossary_template.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+  <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      <meta name="generator" content="PhpSpreadsheet, https://github.com/PHPOffice/PhpSpreadsheet">
```
Comment (mattys101) [inline maintenance/scripts/glossary_template.html:5]:
```suggestion
```

File: maintenance/scripts/glossary_template.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      <title>Untitled Spreadsheet</title>
+      <meta name="author" content="Unknown Creator" />
+      <meta name="title" content="Untitled Spreadsheet" />
+      <meta name="company" content="Microsoft Corporation" />
```
Comment (mattys101) [inline maintenance/scripts/glossary_template.html:9]:
```suggestion
      <meta name="author" content="IOF Maintenance Ontology Working Group" />
      <meta name="title" content="IOF MRO Glossary" />
      <meta name="company" content="IOF/OAGi" />
```
I would make this match what is in the ontology file. Either define it here or replace it with the script. Just a suggestion.

File: maintenance/scripts/glossary_generation.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    now = datetime.datetime.now()
+    return now.strftime("%B %d, %Y")
+
+
+def construct_html_row(row, index):
+    entity_name = row[0]
+    definition = row[1]
+
+    return f'''<tr class="row{index}">
+        <td class="column0 style0 s">&nbsp;{entity_name}</td>
```
Comment (mattys101) [inline maintenance/scripts/glossary_generation.py:82]:
```suggestion
        <td class="column0 style0 s">{entity_name}</td>
```
Not sure we need this non-breaking space.

**Review by mattys101 — APPROVED** [review]
Don't have make the changes but I highly recommend them and regen the glossary page.
Since we are generating it like this, my comment on the README updates pull request are likely invalid regarding where the glossary is sitting.

**Pull request merged**